template/build/webpack.prod.conf.js

```diff
@@ -1,4 +1,5 @@
 var path = require('path')
+var glob = require('glob-all')
```

Comment:
Do we not have a way to do globbing already that doesn't need an additional dependency?